# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/spiralsynthmodular/param.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C.org	Mon Jan 20 14:18:44 2003
+++ ./SpiralSound/Plugins/MeterPlugin/MeterPluginGUI.C	Mon Jan 20 14:19:05 2003
@@ -56,7 +56,7 @@
    MaxBox->box (FL_ENGRAVED_BOX);
    MaxBox->color (16);
    add (MaxBox);
-   Meter = new Fl_VU_Meter (2, 62, 226, 20);
+   Meter = new Fl_VU_Meter (2, 62, 226, 20, NULL );
    Meter->color (FL_BLACK);
    cb_Reset_i (Reset, NULL);
    add (Reset);
